model/registry/process.yaml

```diff
@@ -76,3 +76,38 @@ groups:
           An additional description about the runtime of the process, for example
           a specific vendor customization of the runtime environment.
         examples: 'Eclipse OpenJ9 Eclipse OpenJ9 VM openj9-0.21.0'
+  - id: registry.process.attributes
```

Comment:
Why isn't this nested together with the rest of attributes here? At the top there's already 

```
  - id: registry.process
    prefix: process
```

You can define `cpu.state` already below the others, no?

Reply:
The above group has type `resource`, so I thought I had to make a separate group for `attribute_group`. Is that not the case?

Reply:
That was a temp fix because of broken build-tools. 
There is an open [PR](https://github.com/open-telemetry/semantic-conventions/pull/702) to revert that change

Reply:
With above mentioned PR merged you could define new attribute group under top process

Reply:
Let's try to push to get that one merged, so we can move forward with this one 👍

Reply:
We found out we are still blocked by the resource/group type issue with the build tools. Folks are working on it  hopefully soon we can unblock this

Reply:
It should now be unblocked with #820